doc/connectivity/bluetooth/overview.rst

```diff
@@ -93,3 +101,24 @@ Bluetooth stack.
     * Local controller support as a virtual HCI driver
 
   * Verified with multiple popular controllers
+
+* LE Audio (Auracast) in Host and Controller
+
+  * Full support for Isochronous channels
+  * Generic Audio Framework
+
+    * Basic Audio Profile
+
+      * Unicast server and client
+      * Broadcast source and sink
+      * Broadcast assistant
+      * Scan delegator
+
+    * Common Audio Profile
+
+      * Acceptor
+      * Initiator and Commander
```

Comment:
The initiator is in PR, and Commander is not yet started

Reply:
Thanks